src/components/bcros/businessDetails/LinkActions.vue

```diff
@@ -84,6 +89,11 @@ const actions: ComputedRef<Array<Array<MenuActionItem>>> = computed(() => {
   return [allowedActions]
 })
 
+const redirectToDigitalCredentialsPage = () => {
```

Comment:
can you move this redirect to the redirect composable ? :D
As its composable, all the config and store things should work there too.